docs/source/quickstart.rst

```diff
@@ -91,187 +113,206 @@ as follows (this example is also included in ``quickstart/mlflow_tracking.py``):
                 f.write("hello world!")
             log_artifacts("outputs")
 
-    .. code-block:: R
+If you are using a library that supports autologging, but wish to disable it, you may do so by calling ``mlflow.autolog(disable=True)``.
 
-        library(mlflow)
+- For more details on automatic logging, see :ref:`automatic-logging`.
+- For more details on the explicit logging API, see :ref:`tracking_logging_functions`.
 
-        # Log a parameter (key-value pair)
-        mlflow_log_param("param1", 5)
-
-        # Log a metric; metrics can be updated throughout the run
-        mlflow_log_metric("foo", 1)
-        mlflow_log_metric("foo", 2)
-        mlflow_log_metric("foo", 3)
-
-        # Log an artifact (output file)
-        writeLines("Hello world!", "output.txt")
-        mlflow_log_artifact("output.txt")
-
-Viewing the Tracking UI
------------------------
+View MLflow runs and experiments
+-----------------------------------
 
-By default, wherever you run your program, the tracking API writes data into files into a local
-``./mlruns`` directory. You can then run MLflow's Tracking UI:
+Once you've run your code, you may view the results with MLflow's tracking UI. To start the UI, run:
 
 .. code-section::
 
     .. code-block:: shell
 
         mlflow ui
 
-    .. code-block:: R
+And then navigate to http://localhost:5000 in your browser. You will see a page similar to:
 
-        mlflow_ui()
+.. image:: _static/images/quickstart/quickstart_ui_screenshot.png
```

Comment:
Thanks for updating the screenshot! A couple small asks:

1. Can we show some metrics and parameters in the "Table view" screenshot? Right now, it's hard to see that MLflow recorded these.

2. Can we include a second screenshot showcasing the "Chart view" with a couple of charts?

Reply:
For both table and chart, the top-level screen shows the runs, not the metrics/params.

So I need to drill down, which is fine, but: 

For the RandomForestRegressor sample, it does it all in a single pass, so the metrics don't go up over time. So I need to use a screenshot from a different sample, which is fine, but I'll need to write around it a little.